<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVideoUtilModule</class>
 <widget class="qSlicerWidget" name="qSlicerVideoUtilModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="qMRMLCollapsibleButton" name="videoEncodingCollapsibleButton">
     <property name="text">
      <string>Video encoding</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="videoNodeSelector">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLSequenceNode</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nodeSelectorLabel">
          <property name="text">
           <string>Current video sequence:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="codecSelector"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="encodingLabel">
          <property name="text">
           <string>Encoding type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="parameterLabel">
          <property name="text">
           <string>Encoding parameters:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTableWidget" name="encodingParameterTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="encodeButton">
            <property name="text">
             <string>Encode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCollapsibleButton</class>
   <extends>ctkCollapsibleButton</extends>
   <header>qMRMLCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerVideoUtilModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerVideoUtilModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>videoNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
